### DISCUSSION TITLE
How to safely uninstall Magic

### DISCUSSION THREAD

**Discussion by gkamendje** [RTimothyEdwards/magic#269] [Q&A] [answered]
Looks like there is not `uninstall` target in the Magic `Makefile`. I could not find any install manifest file either. I was wondering what the most safe way for uninstalling Magic would be (i.e remove all the files created when I ran `sudo make install`).

**Comment by RTimothyEdwards** [comment, marked as answer]
Assuming a default installation target, just remove the folder `/usr/local/lib/magic` and its contents, and remove the launch script `/usr/local/bin/magic`.

**Reply by gkamendje** [reply]
thanks a lot
